### ISSUE TITLE
No clear instructions how to install on android

### ISSUE THREAD

**Issue by yurivict** [Astonex/Antox#679]
README says "To get Antox on Google Play, join the Google+ Community, and follow the instructions given."

There are no instructions in that Google+ group that can be found easily. Only a bunch of comments and conversations. At least that is what I see from android browser - pages and pages of comments.

Google Play can't even find "Antox" among apps at all.

Could you please make it more straightforward to install? Why not put instructions in README itself?

**Comment by Astonex** [comment]
On that Google+ Community page you should see a pane called "About this community" which has the instructions, as you can see in this picture http://i.imgur.com/c8On5Pz.png

The instructions are written there because you need to join that community as the first step to getting access to Antox.

**Comment by yurivict** [comment]
Yes, this pane on the right shows on desktop browser, but mobile browser only shows the discussion. I can see this "About the community" only after I switch to "desktop view" in mobile browser.

I bet most people will get lost at this point. Poor design of mobile page by google.

**Comment by Bujiraso** [comment]
I can definitely agree that it is easy to get lost as a mobile user. I knew what I was looking for and it still took me ages to find it.

More troublingly, there are several reports that the download link is broken, and I cannot get the download either. Myself and a friend have joined the community, opted in to the beta, and the play link still reports "We're sorry, the requested URL was not found on this server".

Is the download supposed to be still available? What further steps are needed to retrieve it?
